### ISSUE TITLE
Updating from 3.0.4 to 3.0.5; java.lang.RuntimeException: Invalid id 4096 - maximum id range exceeded

### ISSUE THREAD

**Issue by Fatthunder** [Darkosto/SevTech-Ages#1572]
## Issue / Bug
After updating from 3.0.4 to 3.0.5 on my server I am getting the error:
https://pastebin.com/e89T7S95

I have retried updating with the same results. I believe that there is a command to fix this, though I was unable to find it.  (Don't really know the right terms for googling it)

## Expected Behavior
I should not get this message, and should only need to confirm removed items between updates.

## Possible Solution
I believe there is a command to fix this, but was not easily findable.

## Steps to Reproduce (for bugs)
1. Shutdown MC server
2. Update the files
3. Update Forge
4. Start new Forge
5. /fml confirm missing items
6. Get above error
7. If you confirm the error, and try to connect the world will just crash

## Client Information
* Modpack Version: 3.0.5
* Java Version: 1.8.0.161 (64 Bit)
* Launcher Used: Twitch Launcher using Jar Launcher
* Server/LAN/Single Player: Server
* Resourcepack Enabled?: Soartex Fanver - Modded
* Optifine Installed?: yes

## Server Information
* Java Version: 1.8.0.161 (64 Bit)
* Operating System: Windows Server 2016 Standard Edition
* Hoster/Hosting Soultion: N/A

**Comment by Darkosto** [comment]
Have you added mods to the pack?

**Comment by Fatthunder** [comment]
No,

No mods have been added to the pack, I have used the server version downloads since they were available.

**Comment by Fatthunder** [comment]
yes on the client side, not on the server side.

**Comment by Darkosto** [comment]
You need to include your entire log file in this thread. I have updated 2 servers since 3.0.0 - 3.0.6 and have not had this issue. I've also confirmed with other server owners that they have not had this issue. It seems to be something specific to your server. Please attach any server logs you have.

**Comment by Fatthunder** [comment]
The first two files are before I ran the /fml confirm on the range exceeded.

[latest.log](https://github.com/Darkosto/SevTech-Ages/files/1886729/latest.log)
[debug.log](https://github.com/Darkosto/SevTech-Ages/files/1886730/debug.log)


The next two files are after I ran /fml confirm

[latest.log](https://github.com/Darkosto/SevTech-Ages/files/1886732/latest.log)
[debug.log](https://github.com/Darkosto/SevTech-Ages/files/1886733/debug.log)

**Comment by Fatthunder** [comment]
I was finally about to get the world to load properly. Below is a detailed walked through of how I determined the problem and implemented the solution.

Problem: All block Ids were used up.

Cause: Approx. 250 removed blocks were still holding block Ids.

Determination: Looking server start up logs, I noticed that there was an extremely long list of block with the follow - "[Server thread/INFO] [FML]: Registry Block: Found a missing id from the world". Upon some googling I determined that this message is for removed blocks. (List of my removed blocks: https://pastebin.com/3ikL1AYJ     - The / at the end of the lines are there just so I could track which items were already removed.)

Solution: Opened up the level.dat file in NBTExplorer. Searched for the listed blocks and deleted all occurrences of them.  Upon the removal of all block listed blocks the world was able to load successfully.

**Comment by BonaireDreams** [comment]
Just an FYI for anyone having this issue who has added additional mods. We added minecolonies as we enjoy the village that is built over time. After 3.0.6 I got the Invalid id 4096 - maximum id range exceeded error, but was unable to find any logs for missing ID's to remove. My solution was to add NotEnoughIDs mod on server and clients (We only have 3 players so easy to co-ordinate) Worked like a charm.
